protoc-gen-twirp/generator.go

```diff
@@ -468,6 +479,7 @@ func (t *twirp) generateUtils() {
 	t.P(`  }`)
 	t.P(`  req.Header.Set("Accept", contentType)`)
 	t.P(`  req.Header.Set("Content-Type", contentType)`)
+	t.P(`  req.Header.Set("Accept-Encoding", "gzip")`)
```

Comment:
This change makes it such that all clients using this code include the `Accept-Encoding` header.

Reply:
If a server doesn't support gzip, this header will be just ignored.

Reply:
Right, I understand that. However, a feature like this will only be considered if it is _opt-in_, meaning that clients and servers can safely upgrade their version of Twirp without experiencing any change in how Twirp servers encode responses. 

The current implementation is not opt-in because a user may upgrade their Twirp client and experience a change in behavior. This kind of change would require a release of a new major version.

Reply:
How about addind an option to enable compression which is disabled by default?

Reply:
Please submit a proposal for how this feature should be implemented in Twirp as a Github issue

Reply:
Get it